src/diffusers/pipelines/wan/pipeline_wan_i2v.py

```diff
@@ -137,7 +155,7 @@ def __init__(
         self,
         tokenizer: AutoTokenizer,
         text_encoder: UMT5EncoderModel,
-        image_encoder: CLIPVisionModelWithProjection,
+        image_encoder: CLIPVisionModel,
```

Comment:
need to match with here https://huggingface.co/Wan-AI/Wan2.1-I2V-14B-480P-Diffusers/blob/main/model_index.json#L6